components/Sections/NewFooter/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { FormattedMessage } from 'react-intl';
+import { FaGithub, FaSlack, FaTwitter, FaMastodon } from 'react-icons/fa';
+import styles from './index.module.scss';
+import LocalizedLink from '../LocalizedLink';
```

Comment:
You forgot to update this import. Build is failing. Remember to always run lint commands before pushing ;)